docs/api-overview.md

```
@@ -1808,10 +1832,11 @@ _Appears in:_
 
 | Field | Description | Default | Validation |
 | --- | --- | --- | --- |
-| `phase` _string_ |  |  |  |
-| `observedGeneration` _integer_ | The generation observed by the resource controller. |  |  |
-| `conditions` _[Condition](#condition) array_ |  |  |  |
-| `releases` _[ComponentRelease](#componentrelease) array_ |  |  |  |
+| `phase` _[Phase](#phase)_ | Phase is the top-level lifecycle phase of the module. |  |  |
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)